lib/evss/dependents/retrieved_info.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require 'common/models/redis_store'
+require 'common/models/concerns/cache_aside'
+require_relative './retrieve_info_response'
```

Comment:
these aren't necessary when we are using eagerload/autoload

Reply:
Cool.  Then should I remove this comment line?  https://github.com/department-of-veterans-affairs/vets-api/blob/5ea0f7db414f2198b176258339f134f909874b94/lib/common/models/concerns/cache_aside.rb#L9

Reply:
yea, eager loading loads everything in the eager load paths on startup iirc

Reply:
That's my understanding too, which makes me wonder why the comment I linked to was added in the first place.  (It was added when autoloading of the `lib` directory was enabled in `config/application`)